ScreenControl_Web/src/Connection/ConnectionManager.ts

```diff
@@ -0,0 +1,112 @@
+
+import { MessageReceiver } from "./MessageReceiver";
+import { ServiceConnection } from "./ServiceConnection";
+import { ClientInputAction } from "../ScreenControlTypes";
```

Comment:
This formatting is different to the others, is there a standard we should follow?

Reply:
Is it? There should be consistent formatting that looks like this

'''
import { singlething } from "./allOnOneLine"
import {
    thing1,
    thing2,
    thing3
} from "./multipleThingsOnMultipleLines"

Reply:
Cool, as long as that is the consistency, I am happy